src/CryptorWorker.ts

```diff
@@ -0,0 +1,28 @@
+import { expose } from "threads"
+import * as crypto from 'crypto'
+
+let _cipher: crypto.Cipher
```

Comment:
Why are these using the `let` declaration? Everything at module namepace should be `const`. We only use `let` in code we intend to reassign later.

Reply:
as far as I know, this is just not possible with web workers, we have to pass algo/key/IV in order to recreate the same cryptor in the web workers context. I am unaware of any way to pass custom initialization parameters to threads.js so I've had to resort to a let statement and an `init` function.